src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -803,6 +806,8 @@
     text: az aks update -g MyResourceGroup -n MyManagedCluster --enable-keda
   - name: Disable KEDA workload autoscaler for an existing kubernetes cluster.
     text: az aks update -g MyResourceGroup -n MyManagedCluster --disable-keda
+  - name: Update the SSH key on a AKS Cluster.
+    text: az aks update -g MyResourceGroup -n MyManagedCluster --ssh-key-value <new SSH key value or SSH key file>    
```

Comment:
> -  FAIL - HIGH severity: faulty_help_example_parameters_rule
    Help-Entry: `aks update` - 
	There is a violation:
	"az aks update -g MyResourceGroup -n MyManagedCluster --ssh-key-value <new SSH key value or SSH key file>" is not a valid command.
	unrecognized arguments: --ssh-key-value <new SSH key value or SSH key file>.

```suggestion
    text: az aks update -g MyResourceGroup -n MyManagedCluster --ssh-key-value "<new SSH key value or SSH key file>"    
```

Reply:
@SaurabhSharma-MSFT Could you please resolve this comment by tomorrow? Otherwise, it will not be able to catch up with the release of this sprint.